wp_api/src/wordpress_org/plugin_directory.rs

```diff
@@ -1,8 +1,8 @@
 use super::de::deserialize_default_values;
-use serde::Deserialize;
-use std::{collections::HashMap, fmt::Debug};
+use serde::{Deserialize, Serialize};
+use std::{collections::HashMap, fmt::Debug, sync::Arc};
```

Comment:
Importing `sync::Arc` shouldn't be necessary. See my comment about using `std::sync::Arc`.